src/communication/README.md

```diff
@@ -146,3 +146,35 @@ az communication chat message receipt list --thread "19:xxxxxx" --skip "5"
 ```
 az communication chat message receipt send --thread "19:xxxxxx" --message-id "1xxxxxxxxxxxx"
 ```
+##### Get-Room #####
+```
+az communication rooms get --room "roomId"
```

Comment:
Our command signature usually does not use the plural of nouns
```suggestion
az communication room get --room "roomId"
```

Reply:
Thank you, Xing, for the review. I understand, and we have used singular form for other command groups under communication, but in this specific case, the service itself is called rooms and I think it is better to stay with the plural form "rooms" for consistency.

Reply:
I put a comment here to keep this conversation open and gather more feedback.

Reply:
Thanks for raising this @zhoxing-ms! In this particular case, the use of the plural form may be justified. 

For better or worse, the name of the service that is using the plural form. The same plural form is used in external documentation, APIs, SDKs and more. Here is the [quick start documentation](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/rooms/get-started-rooms) for reference. 

In this case, as service owners, we would prefer to have consistent naming across various mediums, including the CLI, in the interest of making it easy for developers to form associations.

CC: @mayssamm

Reply:
OK, I can accept this reason. Let's see the opinions from assignees @kairu-ms @jsntcy